src/old/provider/MobileProvider/ScreenProvider.tsx

```diff
@@ -4,8 +4,8 @@ import { ScreenContext } from './ScreenContext';
 
 export const ScreenProvider: React.FC = ({ children }) => {
   const theme = useTheme();
-  const mobile = useMediaQuery(theme.breakpoints.down('sm'));
-  const tablet = useMediaQuery(theme.breakpoints.down('md'));
+  const mobile = !useMediaQuery(theme.breakpoints.up('sm'));
```

Comment:
It is a convention in JS to use `is`, `has`, etc. prefixes for boolean values.

So, `isMobile` should be better.

Reply:
Also, what was the problem with using `.down('sm)` ?

Reply:
it should works like on this image
![Screenshot_6](https://user-images.githubusercontent.com/71778107/131849325-f6eaaf85-0910-4750-884b-41c79b1b00ce.png)


but with this solution it works like this
![Screenshot_3](https://user-images.githubusercontent.com/71778107/131849480-195e59dc-d86e-4c00-8f44-255f0beb3c60.png)